src/modules/user/user.controller.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Controller,
+  Put,
+  Body,
+  Req,
+  HttpCode,
+  HttpStatus,
+  Get,
+} from '@nestjs/common';
+import { UserService } from './user.service';
+import UpdateUserDto from '@modules/user/update-user.dto';
+import { ResponsePayload } from '@common/type/response.interface';
+import { Request } from 'express';
+import { AuthType } from '@common/type/auth-type.enum';
+import { Auth } from '@common/decorator/auth.decorator';
+
+@Controller('api')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+
+  @Auth(AuthType.ACCESS_TOKEN)
+  @Put('user')
+  @HttpCode(HttpStatus.OK)
+  async updateCurrentUser(
+    @Req() req: Request,
+    @Body('user') dto: UpdateUserDto,
```

Comment:
```suggestion
    @Body('user') <change_name?>: UpdateUserDto,
```
anh nghĩ nên đặt tên biến rõ nghĩa hơn, `dto` khá chung chung, có thể gây nhầm lẫn

Reply:
anh thấy em update ở service rồi, tuy nhiên ở controller cũng nên rename nhé